src/System.Private.CoreLib/shared/System/Threading/ThreadStartException.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
I guess you didn't annotate `Exception` as nullable in the constructor because it is internal, and internally we shouldn't allow null innerException?

Reply:
Exactly.